cmux-tui/bindings/python/cmux/resources.py

```
@@ -1664,6 +1708,83 @@ def _event_item(value: Any) -> SessionEvent:
     return Unknown(kind, dict(payload))
 
 
+def _journal_record(value: Any) -> SessionJournalRecord:
```

Ruff (0.16.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)